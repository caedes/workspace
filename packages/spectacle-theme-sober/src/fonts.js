export default {
  primary: { name: 'Roboto', googleFont: true, styles: ['400', '700'] },
  secondary: { name: 'Comfortaa', googleFont: true, styles: ['400', '700'] },
};
