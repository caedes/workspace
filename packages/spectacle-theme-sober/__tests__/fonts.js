import fonts from '../src/fonts';

describe('fonts', () => {
  it('returns the correct snapshot', () => {
    expect(fonts).toMatchSnapshot();
  });
});
