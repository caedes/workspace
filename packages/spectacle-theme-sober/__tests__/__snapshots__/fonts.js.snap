// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fonts returns the correct snapshot 1`] = `
Object {
  "primary": Object {
    "googleFont": true,
    "name": "Roboto",
    "styles": Array [
      "400",
      "700",
    ],
  },
  "secondary": Object {
    "googleFont": true,
    "name": "Comfortaa",
    "styles": Array [
      "400",
      "700",
    ],
  },
}
`;
